### ISSUE TITLE
Invite form instructions missing on some sites

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/pantheon-upstream-express-production#11]
When I was inviting someone to the Digital Accessibility site I saw the instructions were missing from the top of the form(https://www.colorado.edu/digital-accessibility/admin/people/invite)

![image](https://user-images.githubusercontent.com/349713/181068044-0e736b81-153d-46a6-ae5f-4c1c8e712f35.png)

However, checking it against the Digital Sign site, those instructions do exist on the form (https://www.colorado.edu/service/digital-signage/admin/people/invite)

![image](https://user-images.githubusercontent.com/349713/181068275-66e58172-5dec-41fe-b466-dcb64517bc85.png)
